### ISSUE TITLE
Add custom shortcut for Elements List - Gesture commands

### ISSUE THREAD

**Issue by luce-carevic** [nvaccess/nvda#17739]
### Is your feature request related to a problem? Please describe.

As I cannot use the Insert key for the NVDA key, I'm using custom shortcuts as a workaround. 
However, It seems there is no way to add a custom shortcut for the Elements List dialog in the Gesture commands panel.

### Describe the solution you'd like

Add the possibility to add a custom shortcut for the Elements List dialog.

**Comment by luce-carevic** [comment]
Thank you so much for your help! I did not know the Input gestures dialog is sensitive to the context.

I cannot change my NVDA key with something that works because I'm using it with a VM on macOS. So my workaround was to add custom shortcuts.

Sorry for the noise and thanks again!
